example/cfg/config_evm-evm_1.json

```diff
@@ -8,7 +8,8 @@
         "path": ".",
         "url": "test.url",
         "encryptionKey": "test-enc-key"
-      }
+      },
+      "commHealthCheckInterval": "24h"
```

Comment:
?

Reply:
no need for health check running in local setup / e2e tests